dwave_networkx/drawing/zephyr_layout.py

```diff
@@ -192,87 +190,84 @@ def draw_zephyr(G, **kwargs):
         only these couplers are displayed.
 
     show_labels: boolean (optional, default False)
-        If True, each chain in ``emb`` is labelled with its key.
+        If True, each chain in the ``emb`` parameter is labelled with its key.
 
     chain_color : dict (optional, default None)
-        Colors as a dict of form {node: rgba_color, ...} associated with
-        each key in ``emb``, where colors are length-4 tuples of floats
-        between 0 and 1 inclusive. If None, each chain is assigned a
+        Colors as a dict of the form ``{node: rgba_color, ...}`` associated with
+        each key in the ``emb`` parameter, where colors are length-4 tuples of floats
+        between 0 and 1, inclusive. If None, each chain is assigned a
         different color.
 
     unused_color : tuple (optional, default (0.9,0.9,0.9,1.0))
-        Color for nodes of G that are not part of chains, and edges
+        Color for graph ``G``'s nodes that are not part of chains and edges
         that are neither chain edges nor interactions. If None, these
         nodes and edges are not shown.
 
     overlapped_embedding: boolean (optional, default False)
-        If True, chains in ``emb`` may overlap (contain the same vertices
-        in G), and these overlaps are displayed as concentric circles.
+        If True, chains in the ``emb`` parameter may overlap (contain the same vertices
+        in ``G``), and these overlaps are displayed as concentric circles.
 
     kwargs : optional keywords
-       See :func:`~networkx.drawing.nx_pylab.draw_networkx` for a description of
-       optional keywords, with the exception of the ``pos`` parameter, which is
-       unsupported. If the ``linear_biases`` or ``quadratic_biases`` parameters
-       are provided, any provided ``node_color`` or ``edge_color`` arguments are
-       ignored.
+       Parameters in :func:`~networkx.drawing.nx_pylab.draw_networkx`, except for the ``pos`` parameter.
+       If the ``linear_biases`` or ``quadratic_biases`` parameters are specified,
+       the :func:`~networkx.drawing.nx_pylab.draw_networkx` ``node_color``
+       or ``edge_color`` parameters are ignored.
     """
     draw_embedding(G, zephyr_layout(G), *args, **kwargs)
 
 def draw_zephyr_yield(G, **kwargs):
-    """Draws the given graph G with highlighted faults, according to layout.
+    """Draws the given graph ``G`` with highlighted faults, according to the Zephyr layout.
 
     Parameters
     ----------
     G : NetworkX graph
-        Graph to be parsed for faults.
+        :term:`Graph` to be parsed for faults.
 
     unused_color : tuple or color string (optional, default (0.9,0.9,0.9,1.0))
-        The color to use for nodes and edges of G which are not faults.
-        If unused_color is None, these nodes and edges will not be shown at all.
+        Color to use for nodes and edges of ``G`` which are not faults.
+        If None, these nodes and edges will not be shown.
```

Comment:
```suggestion
        If None, these nodes and edges are not shown.
```

Adding a second Suggested Change so you can see how you can collect all such suggestion into a single commit